README.md

```diff
@@ -135,6 +145,20 @@ resource "nuodbaas_database" "db" {
 }
 ```
 
+### Connecting to the database
+
+The example configuration above also exposes an [`output`](https://developer.hashicorp.com/terraform/language/values/outputs) named `nuosql_args`, which makes available the arguments to supply to the `nuosql` tool in order to establish a client connection to the database.
+The `nuosql` tool can be found in the [NuoDB Client Package](https://github.com/nuodb/nuodb-client).
+
+Once `nuosql` is installed and available on the system path, you can connect to the database by running the command:
+
+```bash
+eval "nuosql $(terraform output -raw nuosql_args)"
+```
+
+> [!NOTE]
+> If you are running a local instance of the NuoDB Control Plane, then you may have to omit the `--connection-property verifyHostname=true` argument from the `nuosql_args` output, because DNS may not resolve the `sql_endpoint`.
```

Comment:
"... then you may have to omit the `--connection-property verifyHostname=true` argument ..." -> "... then you should specify `--connection-property verifyHostname=false` argument ..."